data/random-teams.ts

```diff
@@ -1129,8 +1126,6 @@ export class RandomTeams {
 					rejectAbility = !counter['Grass'];
 				} else if (ability === 'Own Tempo') {
 					rejectAbility = isDoubles;
-				} else if (ability === 'Power Construct') {
-					rejectAbility = true;
```

Comment:
I mean, this was a placeholder for when it is released with DLC2 and it'd be rejected based on certain conditions (for example: 10% would have an Aura Break set).